ffac-update-location-gps/files/etc/hotplug.d/tty/10-update-location-gps

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+lockpath="/var/lock/hotplug-update-location-gps_${DEVNAME}.lock"
+
+if [ "${ACTION}" = "add" ]; then
+	enabled=$(uci get update-location-gps.settings.enabled)
+	[ "${enabled}" != "1" ] && exit 0
+	test -e "${lockpath}" && exit 0
+	echo "hotplug-update-location-gps: TTY device ${DEVNAME} was plugged in" > /dev/kmsg
```

Comment:
Why are we using kmsg instead of logger here?

Reply:
I consulted gluon IRC as my logging did not show up for logging on boot, as /etc/hotplug.d is loaded before the logger.
Therefore I am logging to /dev/kmsg which shows up in `dmsg`